### ISSUE TITLE
Help needed?

### ISSUE THREAD

**Issue by diego351** [manosim/django-rest-framework-docs#141]
Does anybody still care about the management of pull requests? Do you guys need help?
Project is truly awesome, missing few features but still. It would shame to waste that potential.

**Comment by manosim** [comment]
Hello @diego351! I am Manos and I am the author of drfdocs. Unfortunately due to work and other projects, I don't have enough time to maintain this project the right way.

I would be more than happy to invite more collaborators - and of course help them - if anyone is willing to help and maintain this project as long as there is a plan since there are many PRs waiting to be merged and looks like lots of people are using/depending on it.

If anyone is interested feel free to ping me here. Thank you for your patience with these PRs!

**Comment by diego351** [comment]
I'm partly sure I'm in since I might have more time next month, gonna be sure soon. I'll be happy to help you with the project since I use it heavily on one of my personal ones - I save unbelievable amount time. I'll keep you posted!

**Comment by manosim** [comment]
That sounds great. Let me know whenever you want to get involved and we can come up with a roadmap!

If anyone else is also interested let me know!

**Comment by heyyeah** [comment]
Yes -- and I think there are a few people in other threads who expressed support too.

**Comment by RaitoBezarius** [comment]
Hey, just wanted to chime in to say again : this is really awesome.

I can support sorting through PRs and reviewing them to offload some work @manosim :).

Keep up the great work :+1: !

**Comment by iamhssingh** [comment]
You all are still alive, right? @manosim @heyyeah @diego351 @RaitoBezarius

**Comment by diego351** [comment]
Yes, I'm still here, what's up?
